### ISSUE TITLE
Client is broken

### ISSUE THREAD

**Issue by RubenVerborgh** [LinkedDataFragments/Client.Java#16]
The example in the README

```Java
LinkedDataFragmentGraph ldfg = new LinkedDataFragmentGraph("http://data.linkeddatafragments.org/dbpedia");
Model model = ModelFactory.createModelForGraph(ldfg);
```

fails with

```
Exception in thread "main" java.lang.NullPointerException
                at java.nio.CharBuffer.wrap(CharBuffer.java:487)
                at com.github.fge.uritemplate.parse.URITemplateParser.parse(URITemplateParser.java:55)
                at com.github.fge.uritemplate.URITemplate.<init>(URITemplate.java:52)
                at org.linkeddatafragments.model.LinkedDataFragment.setTemplate(LinkedDataFragment.java:140)
                at org.linkeddatafragments.model.LinkedDataFragment.hydraParse(LinkedDataFragment.java:85)
                at org.linkeddatafragments.model.LinkedDataFragment.<init>(LinkedDataFragment.java:77)
                at org.linkeddatafragments.model.LinkedDataFragmentFactory.create(LinkedDataFragmentFactory.java:18)
                at org.linkeddatafragments.client.LinkedDataFragmentsClient.getBaseFragment(LinkedDataFragmentsClient.java:60)
                at org.linkeddatafragments.model.LinkedDataFragmentGraph.graphBaseSize(LinkedDataFragmentGraph.java:102)
                at com.hp.hpl.jena.graph.impl.GraphBase.size(GraphBase.java:364)
                at org.linkeddatafragments.solver.ReorderTransformationLDF.<init>(ReorderTransformationLDF.java:41)
                at org.linkeddatafragments.model.LinkedDataFragmentGraph.<init>(LinkedDataFragmentGraph.java:37)
```

**Comment by laurensdv** [comment]
@RubenVerborgh 

Try the following fix:

https://github.com/laurensdv/Client.Java/commit/18bb91749db5f6747e2372ac7da60197323e41c7

**Comment by RubenVerborgh** [comment]
@laurensdv While that fix is likely also necessary, applying it still gives the same error.

I also see:
```
 WARN [main] (ErrorHandlerFactory.java:77) - Only triples or default graph data expected : named graph data ignored
```

So I'm suspicious that the client actually gets back TriG or something instead of Turtle.

**Comment by laurensdv** [comment]
Issue might be more tricky than that then, actually Gerald fixed many issues in the java8 branch already.

Maybe consider to make this the default one?

**Comment by RubenVerborgh** [comment]
Certainly! It has not been pushed to GitHub though, and I couldn't find Gerald on public GitHub. Can you follow up on this one?

**Comment by laurensdv** [comment]
There is an open pull request on that.

**Comment by RubenVerborgh** [comment]
You're right, assigned you to #14.

**Comment by RubenVerborgh** [comment]
Fixed in #17.
